.github/workflows/code-review.yaml

```
@@ -0,0 +1,45 @@
+# This workflow will perform the code review on each code change.
+# It uses "actions/cache" and turborepo in order to perform tasks only on changed files.
+
+name: Code Review
+
+on:
+  push:
+    branches: ["main"]
+  pull_request:
+    types: [opened, synchronize]
+
+jobs:
+  build:
+    name: Build and Test
+    timeout-minutes: 5
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Check out code
+        uses: actions/checkout@v3
```

You need to use the commit hash

```suggestion
        uses: actions/checkout@bf085276cecdb0cc76fbbe0687a5a0e786646936
```

---

Why?

---

When using third-party actions, pin the version with a commit hash rather than a tag to shield your workflow from potential supply-chain compromise.